src/mlpack/core/data/load.hpp

```diff
@@ -290,6 +290,51 @@ bool Load(const std::string& filename,
           format f = format::autodetect,
           std::enable_if_t<HasSerialize<T>::value>* = 0);
 
+/**
+ * This function a set of several dataset files into one matrix.
```

Comment:
```suggestion
 * This function loads a set of several dataset files into one matrix.
```
Forgot the verb :smile: